src/main/java/org/opensearch/neuralsearch/search/HybridTopFieldDocSortCollector.java

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.search;
```

Comment:
can we move all collector related classes to package org.opensearch.neuralsearch.search.collector